dsl/src/main/scala/com/crobox/clickhouse/dsl/Query.scala

```diff
@@ -3,7 +3,7 @@ package com.crobox.clickhouse.dsl
 import scala.util.Try
 
 trait Table {
-  val database: String
+  def database: String = ""
```

Comment:
Just curious, why did you instantiate this to an empty string? In the previous scenario compile errors would popup if not set. Now there might be a chance that an empty database is selected (which should be avoided imho)